### DISCUSSION TITLE
Automatically enable C++17 intellisense

### DISCUSSION THREAD

**Discussion by RicardRC** [GodotECS/godex#220] [Ideas & Proposals]
The correct intellisense configuration is the following:
![imagen](https://user-images.githubusercontent.com/35220750/125320881-6b240e00-e33c-11eb-9694-44c482e1f091.png)

Maybe this is a godot "bug" instead of godex?

**Comment by AndreaCatania** [comment]
I think this isn't a bug but rather a feature that allow to specify the C++ version to use and so properly generate the project. Visual Studio Code in Linux has the same issue and it's required to set the C++17 flag to activate intellisense correctly.

**Comment by AndreaCatania** [comment]
Moving the proposal to discussions to keep the issue just for things to fix.
